src/org/labkey/remoteapi/announcements/TestAnnouncementModel.java

```diff
@@ -0,0 +1,263 @@
+package org.labkey.remoteapi.announcements;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.labkey.test.util.WikiHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// POJO for working with responses from Announcements APIs.
+// This is modeled after AnnouncementModel on the server.
+public class TestAnnouncementModel
```

Comment:
I was a little confused by the name of this class when I saw it, especially since it is exposed in the remote API. Would it be too confusing to call it AnnouncementModel as well?

Reply:
Renamed.